portalnet/src/discovery.rs

```diff
@@ -1,5 +1,8 @@
-use super::types::messages::{PortalnetConfig, ProtocolId};
-use crate::socket;
+use std::hash::{Hash, Hasher};
+use std::net::Ipv4Addr;
+use std::str::FromStr;
+use std::{convert::TryFrom, fmt, fs, io, net::SocketAddr, sync::Arc};
```

Comment:
I wonder if there's a clippy warning or something we can add to just have this done once in the code base and then not have to trickle it in over time. (I'm definitely happier with this python-like grouping of library imports at the top: system, 3rd-party, and local)

Are you doing this manually, or using some tool to reorder?

Reply:
https://github.com/rust-lang/style-team/issues/131#issuecomment-995316504

https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#group_imports

I've just been doing this manually (following the python-like grouping) whenever I'm working on a file that I notice doesn't conform. Agree that this isn't ideal, but it also looks like the `rustfmt` lint is only available on `nightly`. 

What do you think? I could go through the codebase, and update locally using the `rustfmt` lint. But I'm not sure we have a solid way forward to prevent deterioration over time.

Reply:
Yeah, I think the status quo is fine for now.